app/controllers/movies_controller.rb

```diff
@@ -7,8 +7,9 @@ def index
   def show
     @movies = Movie.all
     @movie = Movie.find(params[:id])
-    @get_api = HTTParty.get('http://www.omdbapi.com/?t=' + @movie.title + '&apikey=' + ENV['MOVIE_DB_API'])
-    @get_api.gsub!(' ', '+')
+    @movie_string = 'http://www.omdbapi.com/?t=' + @movie.title + '&apikey=' + ENV['MOVIE_DB_API']
+    @movie_string.gsub!(' ', '+')
```

Comment:
Måske er det her en endnu bedre løsning: https://stackoverflow.com/questions/17375947/parsing-string-to-add-to-url-encoded-url